Pharmacy Database Project-csharp version/Console Application/ServiceConfiguration.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AutoMapper;
+using AutoMapper.Extensions.ExpressionMapping;
+using Domain.Models;
+using Domain.Repositories.Interface;
+using Domain.Services.Implementations;
+using Domain.Services.Interfaces;
+using Infrastructure;
+using Infrastructure.Entities;
+using Infrastructure.Repository;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Console_Application;
+
+public static class ServiceConfiguration
+{
+    public static ServiceProvider ConfigureServices()
+    {
+        var serviceCollection = new ServiceCollection();
+
+        //serviceCollection.AddLogging(configure => configure.AddConsole());
+
+        //serviceCollection.AddScoped<ICustomerService, CustomerService>();
+
+        //serviceCollection.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
+
+        // TODO: manage the DbContext lifetime appropriately
+        serviceCollection.AddDbContext<PharmacyDbContext>(options =>
+            options.UseSqlServer("Server=localhost;Database=master;Trusted_Connection=True;Database=Pharmacy;"));
+        serviceCollection.AddScoped<IRepository<WorkHoursDomain, int>, Repository<WorkHours, WorkHoursDomain, int>>();
```

Comment:
based on the comment below "The one that talks about Generic Repo", please register the Repos as follow:

```suggestion
        serviceCollection.AddScoped<IWorkHoursRepository, WorkHoursRepository>();
```